packages/terra-time-input/tests/nightwatch/TimeInputTests.jsx

```diff
@@ -14,8 +14,6 @@ const TimeInputTests = () => (
       <li><Link to="/tests/time-input-tests/twelve-hour-filled-morning">TimeInput - Twelve Hour Default Time Morning</Link></li>
       <li><Link to="/tests/time-input-tests/twelve-hour-filled-evening">TimeInput - Twelve Hour Default Time Evening</Link></li>
       <li><Link to="/tests/time-input-tests/twelve-hour-mobile">TimeInput - Twelve Hour Mobile</Link></li>
```

Comment:
I had to combine all the tests into the same component due to race conditions. I noticed that sometimes, componentWillMount would render before componentWillUnmount of the previous component, meaning sometimes the fake mobile mode would be disabled. Thus, I kept it at one component on the same page to ensure this doesn't happen.